data/core/about.cfg

```diff
@@ -588,7 +588,7 @@
     [/entry]
     [entry]
         name = "battlestar"
-        comment = "- some scenery graphics including burnt tent."
+        comment = "— some scenery graphics including burnt tent."
```

Comment:
Maybe just remove the hyphen entirely?

Reply:
Also outside the scope of the PR, but looking at the context again I think I'll remove these changes since it looks like the comments aren't actually rendered to the user.

Reply:
I reverted the comment changes in the credits files.